src/os/filestore/FileStore.cc

```diff
@@ -107,6 +107,7 @@ using ceph::crypto::SHA1;
 #define XATTR_SPILL_OUT_NAME "user.cephos.spill_out"
 #define XATTR_NO_SPILL_OUT "0"
 #define XATTR_SPILL_OUT "1"
+#define __FUNC__ "function:" << __func__ << ", line:" << __LINE__
```

Comment:
@wjwithagen \_\_func\_\_ has no class prefix, and there is no other way to just get className::funName,  \_\_PRETTY_FUNCTION\_\_ is too informative, may be i can trim from \_\_PRETTY_FUNCTION\_\_, so what's your opinion?